### PR TITLE
Replaces only the items property rather than make a copy of the whole object

### PR DESCRIPTION
**Pull request by fernandosouza**

### CODE REVIEW

**Comment by fernandosouza** [comment]
@eduardolundgren, @brunobasto

**Comment by brunobasto** [comment]
LGTM

File: src/Select.js [hunk 1]
```diff
@@ -158,9 +158,8 @@ class Select extends Component {
 	 * @inheritDoc
 	 */
 	prepareStateForRender(data) {
-		return Object.assign({}, data, {
-			items: this.items.map(item => Soy.toIncDom(item))
-		});
+		data.items = this.items.map(item => Soy.toIncDom(item));
+		return data;
```
Comment (fernandosouza) [inline src/Select.js]: Was this what you were concerned about @eduardolundgren? In fact, there was no reason to use Object.assign here once we can only set a new value for any property into `data` object.

Comment (eduardolundgren) [inline src/Select.js]: Yes. It looks fine now. Thanks.

Comment (fernandosouza) [inline src/Select.js]: Hey, @eduardolundgren. I would like to certify sure if your last comment above means an LGTM.

Comment (eduardolundgren) [inline src/Select.js]: Yep. LGTM.

**Pull request merged**